tests/scripts/all.sh

```diff
@@ -34,13 +34,26 @@ MEMORY=0
 SHORT=0
 FORCE=0
 
+OPENSSL="openssl"
```

Comment:
Ok, you mean leave the command line option there as it is, but also make it possible to define all these values through environment variables?